extensions/http/src/main/resources/META-INF/services/org/mule/runtime/core/config/registry-bootstrap.properties

```diff
@@ -1,4 +1,3 @@
-_httpExtListenerConnectionManager=org.mule.extension.http.internal.listener.server.HttpListenerConnectionManager,applyToArtifactType=app/domain
 #_httpMessageProcessorProvider=org.mule.module.http.internal.request.client.HttpConnectorMessageProcessorProvider
```

Comment:
Though this was used by the mule client, I'll remove it and in any case we can reintroduce it if it's necessary for the revamped mule client.